modules/internationalisation/schemas.ts

```diff
@@ -0,0 +1,4 @@
+import { z } from 'zod';
+import { IsoCurrency } from '@modules/internationalisation/currency';
+
+export const isoCurrencySchema = z.nativeEnum(IsoCurrency);
```

Comment:
IsoCurrency schema - I've put this into a separate file from the type so that code which doesn't need Zod doesn't pull it in by default. I'm not 100% sure this is necessary but I was seeing file sizes swelling in CI and this seems to fix it.

Reply:
Really interesting, I'd love to understand why that was happening.